publish-dev-branch.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Check if working directory is clean
```

Comment:
These scripts essentially allow us to push development builds of every package. It currently pushes it to my personal npm under a dev tag, by changing the scope of the packages from noir-lang, but we can modify it to push to noir-lang (under a dev tag too)